app/views/static_pages/formula.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="menu">
+  <a class="tab index tooltip active" href="/index" name="Projects">
+    <img src="<%= asset_path('leaf-white.svg') %>">
+  </a>
+  <a class="tab tooltip" href="/attributes" name="Attributes">
+    <img src="<%= asset_path('edit-white.svg') %>">
+  </a>
+  <a class="tab tooltip" href="/information" name="Information">
+    <img src="<%= asset_path('info-white.svg') %>">
+  </a>
+  <a class="tab logout tooltip" href="/.." name="Log Out">
+    <img src="<%= asset_path('logout-black.svg') %>">
+  </a>
+</div>
+<div class="content">
+  <h1 class="header"> Bestest Ever Formula Tester </h1>
+  <% @projects.each do |project| %>
+    <div class="card">
+      <h1><%= project.name %></h1>
+      <div class="card-content card-no-bottom general-alt">
+        <% project.years.each do |year| %>
+          <table class="test-table">
+            <tr>
+              <th class="attributes"> attributes </th>
+              <th> <%= year.date %> </th>
+            </tr>
+            <% year.data_values.each do |data_value| %>
+              <tr>
+                <td class="type"><%= DataType.find(data_value.data_type_id).name %></td>
```

Comment:
I was referring specifically to the `DataType.find(data_value.data_type_id).name` line in the code, which is a database query. Usually we create instance variables in the controller instead.

As a note, we can avoid the "N+1" query problem you have in this case: http://www.sitepoint.com/silver-bullet-n1-problem/
It won't make your code nicer but it'll reduce the number of db queries by a lot.